demo/src/App.js

```diff
@@ -112,6 +82,25 @@ export default class App extends PureComponent {
       ...propSettings,
       ...this.state
     };
+    const sliders = this.sliders.map(sliderObj => {
+      const Slider = Object.values(sliderObj)[0];
+      const channel = Object.keys(sliderObj)[0];
+      return (
+        <div key={`container-${channel}`}>
+          <p key={`name-${channel}`}>{channel}</p>
```

Comment:
I think you only need key at the top level?

Reply:
Ah, right.